neurobooth_terra/redcap.py

```diff
@@ -357,12 +357,11 @@ def dataframe_to_tuple(df, df_columns, fixed_columns=None,
                 if col.startswith(indicator_column+'___'):
                     mapping[col] = col.split('___')[1]
             except:
-                error_text = f'Could not split column: {col} for indicator column: {indicator_column}\n \
-                                Is the combine-checkbox-option-into-single-column \
-                                under Redcap survey unchecked??'
-                raise IndicatorColumnError(error_text)
+                raise IndicatorColumnError('Could not split column: {col} for indicator column: {indicator_column}')
```

Comment:
it looks like you're swallowing the original exception here. Even though the try block is small, it seems like that could be useful for debugging.

Reply:
Yes, thanks for catching that! I changed it to the following:
except Exception as e:
        raise IndicatorColumnError("message") from e
        
Now the original exception is captured as __context__ and printed along with IndicatorColumnError message the stack trace.